blocks_vertical/sensing.js

```diff
@@ -456,7 +449,7 @@ Blockly.Blocks['sensing_of'] = {
         }
       ],
       "output": true,
```

Comment:
It looks like this block also gets patched in by scratch-gui.

https://github.com/scratchfoundation/scratch-gui/blob/db4b7a0ad9fbb83941b86e17e58e3df3bb48f844/src/lib/blocks.js#L216

Do we want to delete these incrementally as we migrate the categories, or do a big clean up after they're all in?

Reply:
I think a cleanup sounds preferable in the interest of rolling out the existing changes with minimum trouble. Filed #28 to track.